packages/customize-widgets/src/components/sidebar-block-editor/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	BlockEditorProvider,
+	BlockList,
+	BlockSelectionClearer,
+	ObserveTyping,
+	WritingFlow,
+} from '@wordpress/block-editor';
+import {
+	DropZoneProvider,
+	FocusReturnProvider,
+	SlotFillProvider,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import useSidebarBlockEditor from './use-sidebar-block-editor';
+
+export default function CustomizeSidebarBlockEditor( { sidebar } ) {
```

Comment:
nit: The file name is `sidebar-block-editor` but the component name is `CustomizeSidebarBlockEditor`. Might be nice to make em consistent.